.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
```

Comment:
I would remove the `.idea` directory from the commit and add it to the `.gitignore` file. Could you please do that as well?